src/utils/duration.tsx

```diff
@@ -23,7 +34,10 @@ const toHTML = (value, unit, value2 = null, unit2 = null) => (
     </div>
 );
 
-const format = (microseconds, formatter) => {
+const format = (
+    microseconds: number,
+    formatter: any
+): ReactElement<any, any> | string | null => {
```

Comment:
Maybe this return type can be move to the formatter? Both making the return type infered, and removing the any type of the formatter?

Reply:
I see the two formatters in question were above. I guess at this point it's preferable to have, as you wrote, a conservative refactor, and deal with the any types later.